exercises/practice/killer-sudoku-helper/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+  id "java"
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testImplementation platform("org.junit:junit-bom:5.10.0")
+  testImplementation "org.junit.jupiter:junit-jupiter"
+  testImplementation "org.assertj:assertj-core:3.24.2"
```

Comment:
The [java-test-runner](https://github.com/exercism/java-test-runner/blob/main/lib/build.gradle) currently has AssertJ Core 3.15.0. Since the test runner is run without any network connections, that means that all submissions from students also run with that version of AssertJ Core.

To stay safe I think it's best to stick to that version here as well. This way we are sure that the test suite can be run on the test runner.